source/locales.tex

```diff
@@ -3194,48 +3196,48 @@
 \begin{codeblock}
 namespace std {
   template <class charT, class OutputIterator = ostreambuf_iterator<charT>>
-  class num_put : public locale::facet {
-  public:
-    using char_type = charT;
-    using iter_type = OutputIterator;
-
-    explicit num_put(size_t refs = 0);
-
-    iter_type put(iter_type s, ios_base& f, char_type fill, bool v) const;
-    iter_type put(iter_type s, ios_base& f, char_type fill, long v) const;
-    iter_type put(iter_type s, ios_base& f, char_type fill, long long v) const;
-    iter_type put(iter_type s, ios_base& f, char_type fill,
-                  unsigned long v) const;
-    iter_type put(iter_type s, ios_base& f, char_type fill,
-                  unsigned long long v) const;
-    iter_type put(iter_type s, ios_base& f, char_type fill,
-                  double v) const;
-    iter_type put(iter_type s, ios_base& f, char_type fill,
-                  long double v) const;
-    iter_type put(iter_type s, ios_base& f, char_type fill,
-                  const void* v) const;
+    class num_put : public locale::facet {
+    public:
+      using char_type = charT;
+      using iter_type = OutputIterator;
+
+      explicit num_put(size_t refs = 0);
+
+      iter_type put(iter_type s, ios_base& f, char_type fill, bool v) const;
+      iter_type put(iter_type s, ios_base& f, char_type fill, long v) const;
+      iter_type put(iter_type s, ios_base& f, char_type fill, long long v) const;
+      iter_type put(iter_type s, ios_base& f, char_type fill,
+                    unsigned long v) const;
```

Comment:
I think we don't need these next couple of linebreaks. If you remove them now, I will never have to go back to Clause 22 again, which I would appreciate.